docs/eportal/README.md

```diff
@@ -78,183 +73,170 @@ EOL
 EOL
 ```
 
 Install ePortal:
 
-```
-apt-get update && apt-get install -y --no-install-recommends kcare-eportal
+```text
+# apt-get update && apt-get install -y --no-install-recommends kcare-eportal
 ```
 
 ### Ubuntu 22.04
 
 Setup ePortal repo:
 
-```
-apt-get install -y --no-install-recommends curl ca-certificates
+```text
+# apt-get install -y --no-install-recommends curl ca-certificates
 
-curl https://repo.cloudlinux.com/kernelcare/kernelcare.gpg -o /usr/share/keyrings/kcare-eportal.gpg
+# curl https://repo.cloudlinux.com/kernelcare/kernelcare.gpg -o /usr/share/keyrings/kcare-eportal.gpg
 
-cat > /etc/apt/sources.list.d/kcare-eportal.list <<EOL
+# cat > /etc/apt/sources.list.d/kcare-eportal.list <<EOL
 deb [signed-by=/usr/share/keyrings/kcare-eportal.gpg] https://repo.cloudlinux.com/kcare-eportal/22.04 jammy main
 EOL
 ```
 
 Install ePortal:
 
+```text
+# apt-get update && apt-get install -y --no-install-recommends kcare-eportal
 ```
-apt-get update && apt-get install -y --no-install-recommends kcare-eportal
-```
+
 ## Allowed hosts
-We must declare the list of allowed hosts in the configuration for security reasons. It help to avoid HTTP Host header attacks.
-According to your installation, you need to add this parameter into [ePortal config file](#config-files).
+
+We must declare the list of allowed hosts in the configuration for security reasons. It help to avoid HTTP Host header attacks. According to your installation, you need to add this parameter into [ePortal config file](/eportal/#config-files):
+
 ```python
 ALLOWED_HOSTS = ['node1-yourdomain.example.com', 'node2-yourdomain.example.com']
 ```
-Instead of an example, use the DNS name(s) of your ePortal server installation. Deafault value is `[]`.
+
+Instead of an example, use the DNS name(s) of your ePortal server installation. Default value is `[]`.
 
 ## Cache mode
 
-Cache mode allows to greatly reduce disk usage requirements and speed up
-initial downloading step.
+Cache mode allows to greatly reduce disk usage requirements and speed up initial downloading step.
 
-In cache mode ePortal downloads only lightweight meta information about
-patchsets and proxies patch requests from KernelCare agent to the patch server
-(Patch Source). Downloaded patch binaries are cached for 2 weeks and
-accessible for following requests directly from ePortal and doesn't consume
-public internet bandwidth.
+In cache mode ePortal downloads only lightweight meta information about patchsets and proxies patch requests from KernelCare agent to the patch server (Patch Source). Downloaded patch binaries are cached for 2 weeks and accessible for following requests directly from ePortal and doesn't consume public internet bandwidth.
 
 ![Cache mode](/images/eportal-cache-mode.svg)
 
 ### Enable cache mode
 
-1. add
-
-   ```
-   CACHE_MODE = True
-   ```
+1. add the following setting into the [ePortal config file](/eportal/#config-files):
 
-   setting into [ePortal config file](#config-files)
```

Comment:
Snippet would be rendered outside of numbered list item due to zero indentation.